### DISCUSSION TITLE
Is there a way to show an image in composable which has wrapContentSize Modifier?

### DISCUSSION THREAD

**Discussion by ChanJun-Park** [skydoves/landscapist#138] [Q&A] [answered]
In XML, by adding `android:adjustViewBounds="true"` to ImageView, We can make the Image which is loaded with glide wrap image content size.

https://stackoverflow.com/questions/36789526/glide-make-imageview-wrap-content-useless-and-no-animatiion-using-target

I want to do same thing in compose with Landscapist's Glide api.

So, Is there a way to show an image in composable which has wrapContentSize Modifier?

**Comment by yabomonkey** [comment]
I am also looking for a way to load an image via Landscapist's Glide API into a composable which has wrapContentSize set as a modifier. Did you have any luck in achieving this?

**Comment by skydoves** [comment, marked as answer]
Hi everyone, now you can wrap content images by default since version **2.1.0**. Thank you for the reports!
